curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6555231eac4d9f19bd3d44b5.md

```diff
@@ -0,0 +1,75 @@
+---
+id: 6555231eac4d9f19bd3d44b5
+title: Step 69
+challengeType: 20
+dashedName: step-69
+---
+
+# --description--
+
+Calling `vigenere` with `1` to encrypt and `-1` to decrypt is fine but it might be a little bit cryptic. Create a new function called `encrypt` that takes `message` and `key` parameters.
+
+# --hints--
+
+You should declare a new function called `encrypt`. Remember to use the `pass` keyword to fill the body.
+
+```js
+({ test: () => assert(__pyodide.runPython(`
```

Comment:
This is the first time Campers will have heard of the `pass` keyword. So, they should be taught about it.